### DISCUSSION TITLE
Connections counter

### DISCUSSION THREAD

**Discussion by orrans** [felladrin/linkedin-autoconnect-chrome-extension#10] [Ideas]
Can you please add connection counter?
that it will show how many connections its asked after each request.
also maybe to add some option to tell it how much connection you want so if you configure it to 30 it will only press on 30 connections request and stop

**Comment by frazras** [comment]
I agree with this one, LinkedIn triggers its bot detection after about 100 connects. It would be nice to be able to count the connects in the current session and stop after a specified number.

**Comment by felladrin** [comment]
Good point. I'll add it to the roadmap.

**Comment by felladrin** [comment]
This feature was implemented, and now you can set, in the Extension Option screen, the maximum number of connections for the session. It will stop auto-connecting as soon as it reaches the limit.
Please check the [latest release](https://github.com/felladrin/linkedin-autoconnect-chrome-extension/releases/latest) and let me know what you think.
